src/cljs/rems/actions/action.cljs

```diff
@@ -20,10 +20,10 @@
     :data-target (str "#" (action-collapse-id id))}
    (text :t.actions/cancel)])
 
-(defn- action-judge-comment [id label-title comment on-comment]
+(defn action-judge-comment [{:keys [id label comment on-comment]}]
```

Comment:
Kind of  shame that the "old legacy" static workflow component goes into the new namespace.

Reply:
I refer to the term judge here but I like the extracted comment component so perhaps we could combine these but you can do what you like in the next PR.